flags.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The conprof Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/prometheus/common/model"
+	"github.com/thanos-io/thanos/pkg/extkingpin"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func modelDuration(flags *kingpin.FlagClause) *model.Duration {
+	value := new(model.Duration)
+	flags.SetValue(value)
+
+	return value
+}
+
+func regGRPCFlags(cmd extkingpin.FlagClause) (
```

Comment:
All of this was made public on Thanos.. so cannot we use it?

Reply:
Oh, right. I missed this! Updated.